components/CollectiveNavbar.js

```diff
@@ -303,6 +305,20 @@ const CollectiveNavbar = ({
           )}
         </Flex>
         <ExpandMenuIcon onClick={() => setExpended(!isExpended)} />
+        <Container>
+          <FormattedMessage
+            id="Collective.Hero.Host"
+            defaultMessage="{FiscalHost}: {hostName}"
+            values={{
+              FiscalHost: <DefinedTerm term={Terms.FISCAL_HOST} />,
+              hostName: (
+                <LinkCollective collective={collective.host}>
+                  <Span color="black.600">{collective.host.name}</Span>
+                </LinkCollective>
+              ),
+            }}
+          />
+        </Container>
```

Comment:
`collective.host` can actually be `null`

Reply:
@Betree  yes so may be if `!collective.host` then we can render it, or else not needed

But again as you said let's wait for them, I have no issues with waiting :)